runtime-job-worker/pom.xml

```
@@ -51,6 +55,12 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>uk.org.webcompere</groupId>
+      <artifactId>system-stubs-core</artifactId>
+      <version>${system-stubs.version}</version>
```

Let's add test scope here.
```suggestion
      <version>${system-stubs.version}</version>
      <scope>test</test>
```

---

Cf. https://github.com/camunda/connector-sdk/pull/111/commits/30dbc88ae1a50b49783d9f444bde86ce7d0fb5ea.